conformance/service_import.go

```diff
@@ -204,5 +222,50 @@ var _ = Describe("", func() {
 						reportNonConformant("The service ports were not resolved correctly"))
 				})
 		})
+		Context("with conflicting annotations and labels", func() {
+			BeforeEach(func() {
+				helloServiceExport.Spec.ExportedAnnotations = map[string]string{"dummy-annotation": "true"}
+				helloServiceExport.Spec.ExportedLabels = map[string]string{"dummy-label": "true"}
+
+				helloServiceExport2.Spec.ExportedAnnotations = map[string]string{"dummy-annotation2": "true"}
+				helloServiceExport2.Spec.ExportedLabels = map[string]string{"dummy-label2": "true"}
+			})
+
+			Specify("should apply the conflict resolution policy and report a Conflict condition on each ServiceExport",
+				Label(RequiredLabel), func() {
+					AddReportEntry(SpecRefReportEntry, "https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api#labels-and-annotations")
+
+					t.awaitServiceExportCondition(&clients[0], v1alpha1.ServiceExportConflict)
+					t.awaitServiceExportCondition(&clients[1], v1alpha1.ServiceExportConflict)
+
+					serviceImport := t.awaitServiceImport(&clients[0], t.helloService.Name, func(serviceImport *v1alpha1.ServiceImport) bool {
+						return len(serviceImport.Labels) > 0
+					})
+					Expect(serviceImport).NotTo(BeNil(), "ServiceImport was not found")
+
+					Expect(serviceImport.Annotations).To(Equal(helloServiceExport.Annotations), reportNonConformant(""))
+					Expect(serviceImport.Labels).To(Equal(helloServiceExport.Labels), reportNonConformant(""))
+				})
+		})
+	})
+
+	Context("", func() {
+		BeforeEach(func() {
+			helloServiceExport.Spec.ExportedAnnotations = map[string]string{"dummy-annotation": "true"}
+			helloServiceExport.Spec.ExportedLabels = map[string]string{"dummy-label": "true"}
+		})
+
+		Specify("export labels and annotations",
+			Label(OptionalLabel), func() {
```

Comment:
Let's define a specific label as well so this test can be skipped if the implementation doesn't support exported labels.

`const ExportedLabelsLabel = "ExportedLabels"`

```suggestion
			Label(OptionalLabel), Label(ExportedLabelsLabel), func() {
```